### PR TITLE
modify readme.md about pom polishing #508

### PR DESCRIPTION
**Pull request by bttanakagn**
Please review #508.

### CODE REVIEW

**Comment by btyoshikawaa** [comment]
@bttanakagn 
Are not required follow descriptions?

* terasoluna-gfw-common-dependencies
* terasoluna-gfw-validator
* terasoluna-gfw-string
* terasoluna-gfw-codepoints

**Comment by bttanakagn** [comment]
@btyoshikawaa 
I added those artifacts except for `terasoluna-gfw-common-dependencies`, because `terasoluna-gfw-web-dependencies` depends on it.

File: README.md [hunk 1]
```diff
@@ -83,6 +83,12 @@ After adding above, add the following dependency definitions.
 
 <!-- OPTIONAL -->
 <dependency>
+    <groupId>org.terasoluna.gfw</groupId>
+    <artifactId>terasoluna-gfw-codepoints</artifactId>
+</dependency>
```
Comment (btyoshikawaa) [inline README.md]: - [ ] In [guideline](http://terasolunaorg.github.io/guideline/5.1.0.RELEASE/ja/Overview/FrameworkStack.html#terasoluna-gfw-codepoints), codepoints component is shown `terasoluna-gfw-codepoints` only. Let to align to it.

Comment (btyoshikawaa) [inline README.md]: - [ ] Please be arranged to order of `string`, `codepoints`, `validator`.

Comment (bttanakagn) [inline README.md]: @btyoshikawaa 
I removed the descriptions about catalog. 2782cb8ce58269e4578852822c78d6281e0eb7f5.
and arranged the order of `string`,`codepoints`,`validator`. 49af6def209582b432b34602e8f81eea5db7d4d7.

**Pull request merged**